crypto/bn/bn_lib.c

```diff
@@ -640,6 +640,24 @@ int BN_is_bit_set(const BIGNUM *a, int n)
     return (int)(((a->d[i]) >> j) & ((BN_ULONG)1));
 }
 
+int BN_count_low_zero_bits(const BIGNUM *a) 
+{
+    int i, bits;
+    BN_ULONG w;
+
+    bn_check_top(a);
+    for (i = 0; i < a->top; i++) {
+        if (a->d[i] != 0) {
+            bits = 0;
+            for (w = a->d[i]; (w & 1) == 0; w >>= 1)
```

Comment:
I'd much prefer this to use a compiler intrinsic than a loop.  Many compilers won't optimise this efficiently.

E.g. gcc provides __builtin_ctz which could replace this loop.

If there is no intrinsic available, try one of the parallel or multiply/division algorithms like these: https://graphics.stanford.edu/~seander/bithacks.html#ZerosOnRightLinear

Reply:
I agree this could be improved, I mainly wanted to keep the code simple for my first PR